packages/dev/core/src/Engines/Extensions/engine.renderTarget.ts

```diff
@@ -1,4 +1,4 @@
-import { InternalTexture, InternalTextureSource } from "../../Materials/Textures/internalTexture";
+import { HasStencilAspect, InternalTexture, InternalTextureSource } from "../../Materials/Textures/internalTexture";

```

Comment:
Shouldnt we move HasStencilAspect on RenderTargetWrapper insted of internalTexture ?